pySDC/projects/Resilience/accuracy_check.py

```diff
@@ -12,6 +12,10 @@
 from pySDC.projects.Resilience.piline import run_piline
```

Comment:
Looks like changes made here are not part of this PR? If so, please remove.

Reply:
This makes the script much more general for checking the order. I actually took it from another PR, that is true. But I need the changes to supply a custom description, which is where I set the sweeper and I use this script from the test script.